src/subscribr.js

```diff
@@ -55,6 +55,12 @@ module.exports = class Subscribr {
 
 }
 
+let idCounter = 0;
```

Comment:
should we use a closure over here to keep things tidy?
something like 
```
const uniqueId = (() => {
  let idCounter = 0;
  return () => ++idCounter;
})();
```

maybe it's time to create a `utils.js` file for all these auxiliar functions, what do you think?